### PR TITLE
pkg/lib: Fixes for eslint

### PR DESCRIPTION
**Pull request by mvollmer**
These things turn up when running "npm run eslint" or "npm run eslint:fix".

### CODE REVIEW

File: pkg/lib/cockpit-po-plugin.js [hunk 1]
```diff
@@ -24,15 +25,17 @@ module.exports = class {
...
+        var statement;
+        var ret = null;
         const plurals = header["plural-forms"];
 
         if (plurals) {
             try {
                 /* Check that the plural forms isn't being sneaky since we build a function here */
                 Jed.PF.parse(plurals);
-            } catch(ex) {
-                fatal("bad plural forms: " + ex.message, 1);
```
Comment (mvollmer) [inline pkg/lib/cockpit-po-plugin.js:35]: I don't know where `fatal` would come from.  Is it a real thing that we can call?

Comment (marusak) [reply pkg/lib/cockpit-po-plugin.js:35]: It used to be defined in `po/po2json` as:
```
function fatal(message, code) {                                                 
    console.log((filename || "html2po") + ": " + message);                      
    process.exit(code || 1);                                                    
} 
```
See https://github.com/cockpit-project/cockpit/commit/72c180d78848946b939bbab23b063eee246336d8

Comment (martinpitt) [reply pkg/lib/cockpit-po-plugin.js:35]: Oops! Thanks for catching that!

**Review by martinpitt — APPROVED** [review]
Thanks for cleaning up after me! :-)

**Pull request merged**